README.md

```diff
@@ -1,51 +1,94 @@
-# <u>refine.bio Examples Guide: </u>
-This repository contains example workflows of how to use data downloaded from 
+# __refine.bio Examples Guide:__
+This repository contains example workflows of how to use data downloaded from
 [refine.bio](https://www.refine.bio).
 
+## How to use:
+
+If you are comfortable with GitHub, we recommend [cloning](https://help.github.com/articles/cloning-a-repository/)
+this repository and following the examples.
+However, if you are less comfortable with GitHub, you can also use the green
+button that says "Clone or download" to "Download ZIP" and then use these
+notebooks as you please on your own computer.
+Make sure you are on the master branch.
+Which is indicated toward the top of the page, in a gray button that says "Branch:".
+
+In each directory, there is a `README` that explains how to complete the
+analyses with the R or GenePattern notebooks.
+If you are following the instructions for completing the analyses in R, open the
+R markdown (`Rmd`) and follow the instructions.
+You will also see the example output files which are generally TSV or PNG files.
+All the data used in these examples are deposited into the `data/`
+directory and are referenced as such in the R Notebooks.
```

Comment:
I don't believe any of these are R Notebooks

https://github.com/AlexsLemonade/refinebio-examples/blob/cansav09/README_streamlining/clustering/clustering_example.Rmd#L5

Reply:
I changed all notebooks to be nb.html output now.

Reply:
Re-running the output files now and I'll push all those.